src/main/java/com/example/bcsd/Post.java

```diff
@@ -0,0 +1,59 @@
+package com.example.bcsd;
+
+import java.time.LocalDate;
+
+public class Post {
+    protected Integer post_num;
+    protected String title;
+    protected String text;
+    protected LocalDate creation_date;
+    protected LocalDate modification_date;
+
+    public Post() {
+        Integer post_num;
+        String title;
+        String text;
+        LocalDate creation_date;
+        LocalDate modification_date = null;
+    }
```

Comment:
생성자를 표현하신것 같은데 이렇게 되면 필드 초기화가 제대로 되지 않을 것 같아요! 
```suggestion
    public Post(
         Integer post_num,
        String title,
        String text,
        LocalDate creation_date,
        LocalDate modification_date
    ) {
       this.post_num = post_num;
       this.title = title;
       this.text = text;
       this.creation_date = creation_date;
       this.modification_date = modification_date;
    }
```
이렇게 하는건 어떨까요?